src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -154,6 +154,10 @@ function OptionRowLHN(props) {
     const statusContent = formattedDate ? `${statusText} (${formattedDate})` : statusText;
     const isStatusVisible = Permissions.canUseCustomStatus(props.betas) && !!emojiCode && ReportUtils.isOneOnOneChat(optionItem);
 
+    const isGroupChat =
+        optionItem.type === CONST.REPORT.TYPE.CHAT && _.isEmpty(optionItem.chatType) && !optionItem.isThread && lodashGet(optionItem, 'displayNamesWithTooltips.length', 0) > 2;
```

Comment:
Also why are we not using `isGroupChat` here?

Reply:
@stitesExpensify 

> Why is a thread not considered a group chat? There can be multiple participants right? Is it just because we want the title to be different?

Yes, this is correct. We only want to change the title for actual group chats, but not threads.

> Also why are we not using `isGroupChat` here?

I assume you're referring to the `isGroupChat` method in `ReportUtils`? The `optionItem` and `report` are two different objects with completely different properties and structures. The `optionItem` is used only in the `OptionRowLHN` component across the codebase.